compiler/daml-lf-verify/README.md

```diff
@@ -0,0 +1,248 @@
+# DAML Verification Tool
+
+This project performs fully automated formal verification of DAML code.
+Provided with a choice and a field, the verification tool employs symbolic 
+execution and an SMT solver, to verify whether the given choice always preserves
+the total amount of the given field. This allows it to automatically answer
+the common question "Could this smart contract model ever burn money or create
+money out of thin air?"
+
+## Installation
+
+- Install the [DAML development dependecies](https://github.com/digital-asset/daml/).
+
+- Install the [Z3 SMT solver](https://github.com/Z3Prover/z3). 
```

Comment:
Do you still look this up via Bazel runfiles or is it found on PATH?

Reply:
It's looked up via Bazel runfiles, in `Verify.hs`